pytket/extensions/qiskit/qiskit_convert.py

```diff
@@ -855,6 +856,11 @@ def append_tk_command_to_qiskit(
 supported_gate_rebase = AutoRebase(_protected_tket_gates)
 
 
+def _has_implicit_permutation(circ: Circuit) -> bool:
+    impl_dict: dict[Qubit, Qubit] = circ.implicit_qubit_permutation()
+    return tuple(impl_dict.keys()) != tuple(impl_dict.values())
```

Comment:
```suggestion
    return any(q0 != q1 for q0, q1 in circ.implicit_qubit_permuation().items())
```
More efficient because of shortcutting.

Reply:
Thanks